export.js

```diff
@@ -90,43 +181,35 @@ function processTableExport(table) {
             console.log('Finalizing writes into: ' + table + '.json');
             writeStream.end();
         });
-    });
+    }));
 }
 
-systemClient.connect()
-    .then(function (){
-        var systemQuery = "SELECT columnfamily_name as table_name FROM system.schema_columnfamilies WHERE keyspace_name = ?";
-        if (systemClient.metadata.keyspaces.system_schema) {
-            systemQuery = "SELECT table_name FROM system_schema.tables WHERE keyspace_name = ?";
-        }
-
-        console.log('Finding tables in keyspace: ' + KEYSPACE);
-        return systemClient.execute(systemQuery, [KEYSPACE]);
-    })
-    .then(function (result){
-        var tables = [];
-        for(var i = 0; i < result.rows.length; i++) {
-            tables.push(result.rows[i].table_name);
-        }
-
-        if (process.env.TABLE) {
-            var tables = process.env.TABLE.split(',');
-            return Promise.each(tables, function(table){
-                return processTableExport(table);
-            });
-        }
+function getTables() {
+    if (process.env.TABLE)
+        return Promise.resolve(process.env.TABLE.split(','));
 
-        return Promise.each(tables, function(table){
-            return processTableExport(table);
-        });
-    })
-    .then(function (){
+    var systemQuery = "SELECT columnfamily_name AS table_name FROM system.schema_columnfamilies WHERE keyspace_name = ?";
+    if (systemClient.metadata.keyspaces.system_schema) {
+        systemQuery = "SELECT table_name FROM system_schema.tables WHERE keyspace_name = ?";
+    }
+
+    console.log('Finding tables in keyspace: ' + KEYSPACE);
+    return systemClient.execute(systemQuery, [KEYSPACE]).
+        then(result => Array.from(result.rows, row => row.table_name));
+}
+
+systemClient.connect()
+    .then(() => processSharedSchemaExport(KEYSPACE, DIRECTORY + "/_shared.cql"))
```

Comment:
Why `_`?

Reply:
So it can't be confused with any actual table (AFAIK they can't start with _).